frontend/models/data-table.tsx

```diff
@@ -0,0 +1,12 @@
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons"
+
+export interface IncidentData {
+  dates: string
+  incidentType: string
+  officersInvolved: string[]
+  subject: IconDefinition
+  source: IconDefinition
```

Comment:
I'm not sure if we want to update this/our mock data based on the new search flow/wireframes? 
(Totally not necessary if this is a first rendition/placeholder, but just noting for future!)